pkg/ovn_ic_controller/ovn_ic_controller.go

```diff
@@ -571,27 +674,24 @@ func (c *Controller) listRemoteLogicalSwitchPortAddress() (*strset.Set, error) {
 	return existAddress, nil
 }
 
-func (c *Controller) getTSName(index int) string {
-	if index == 0 {
-		return util.InterconnectionSwitch
+func moveElements(arr []string, order int) []string {
```

Comment:
```suggestion
func removeElement(arr []string, order int) []string {
```

Reply:
it renamed it to generateNewOrdergwNodes